### PR TITLE
Move the most recent layer to the top of the layers list

### PR DESCRIPTION
**Pull request by ziw-liu**
Implements #257.

### CODE REVIEW

File: recOrder/plugin/main_widget.py [hunk 1]
```diff
@@ -1021,43 +1024,52 @@ def handle_lc_states_emit(self, value: tuple[tuple, dict[str, list]]):
...
+        cmap : str, optional
+            colormap to render in, by default "gray", use "rgb" for RGB images
+        move_to_top : bool, optional
+            whether to move the updated layer to the top of layers list, by default True
+        """
+        if name in self.viewer.layers:
+            self.viewer.layers[name].data = image
+            if move_to_top:
+                src_index = self.viewer.layers.index(name)
+                self.viewer.layers.move(src_index, dest_index=-1)
```
Comment (talonchandler) [inline recOrder/plugin/main_widget.py:1051]: Nice!

File: recOrder/plugin/main_widget.py [hunk 2]
```diff
@@ -1021,43 +1024,52 @@ def handle_lc_states_emit(self, value: tuple[tuple, dict[str, list]]):
...
-        if "Background Orientation" in self.viewer.layers:
-            self.viewer.layers["Background Orientation"].data = value[1]
-        else:
-            self.viewer.add_image(
-                value[1], name="Background Orientation", colormap="gray"
-            )
+    @Slot(object)
+    def handle_bg_bire_image_update(self, value):
+        self._add_or_update_image_layer(value[0], "Background Retardance")
+        self._add_or_update_image_layer(value[1], "Background Orientation")
```
Comment (talonchandler) [inline recOrder/plugin/main_widget.py:1065]: Possibly out of scope for this PR, but can we add `cmap="hsv"` to make the backgrounds consistent with acquisitions?

File: recOrder/plugin/main_widget.py [hunk 3]
```diff
@@ -1021,43 +1024,52 @@ def handle_lc_states_emit(self, value: tuple[tuple, dict[str, list]]):
...
 
-    @Slot(object)
-    def handle_bg_image_update(self, value):
+    def _add_or_update_image_layer(
+        self,
+        image: NDArray,
+        name: str,
+        cmap: str = "gray",
+        move_to_top: bool = True,
+    ):
```
Comment (talonchandler) [inline recOrder/plugin/main_widget.py:1033]: Excellent.

**Review by talonchandler — APPROVED** [review]
Tested...works extremely well. Only one minor suggestion. Great job!

File: recOrder/plugin/main_widget.py (hunk 2 above)
Comment (ziw-liu) [reply recOrder/plugin/main_widget.py:1065]: Yes that makes sense. Will file new issue and PR.

**Pull request merged**